src/ffmpeg-util.jl

```diff
@@ -268,6 +308,7 @@ function save(path::String, io::VideoStream; video_options...)
         cp(io.path, path; force=true)
     end
     rm(io.path)
+    stop!(io.tick_controller)
```

Comment:
With respect to #3883 I think we just need to move this into the finalizer. Though this whole set up is a bit dodgy because by default, it overwrites tick events from the creation of VideoStream until this call...